types/types.proto

```diff
@@ -165,11 +165,10 @@ message ResponseFlush{
 }
 
 message ResponseInfo {
-	string info = 1; // backwards compatible
-
-	TMSPInfo tmsp_info = 2;
-	LastBlockInfo last_block = 3;
-	ConfigInfo config = 4;
+	string data = 1;
+	string version = 2;
```

Comment:
How does it get lower?  Tendermint should try to get some info from the app, and InfoSync() is how Tendermint gets that info during handshake.